package.json

```diff
@@ -5,7 +5,8 @@
   "repository": "https://github.com/thefrontside/bigtest",
   "scripts": {
     "build": "bin/each-pkg yarn build",
-    "test": "bin/each-pkg yarn test"
+    "test": "bin/each-pkg yarn test",
+    "lint": "bin/each-pkg yarn lint"
```

Comment:
Nice call adding this to the top-level `package.json`!

With this we can add a linting step to our CI. We'll probably want it to come before [this build step](https://github.com/thefrontside/bigtest/blob/master/.circleci/config.yml#L49-L51) of our `build` job.